KomaMRIPlots/src/ui/DisplayFunctions.jl

```diff
@@ -1035,8 +1035,8 @@ function plot_phantom_map(
     max_time_samples=100,
     kwargs...,
 )
-    function interpolate_times(motion)
-        t = times(motion)
+    function interpolate_times(motion, T)
```

Comment:
Same here. You are modifying Base, Core, and Plots. This should be only a change in Base.

Reply:
This modification (obviating the T-parameter problem already solved in [cd8f751](https://github.com/JuliaHealth/KomaMRI.jl/pull/499/commits/cd8f75157b76181b41a3f8a061e9c1cf17cc94a2)) in `plot_phantom_map` came from [this comment](https://github.com/JuliaHealth/KomaMRI.jl/pull/495#issuecomment-2413961563) on the phantom documentation PR. I thought it won't be a bad idea to solve it in this PR, even changing KomaMRIPlots.

Regarding de changes in core, the only change is the removal of the adapt function for NoMotion in Functors.jl, since it is no longer necessary

Reply:
Anyway, I can leave the change of the plot to the other PR.